pkg/provider/cloud/kubevirt/csi.go

```diff
@@ -0,0 +1,163 @@
+package kubevirt
+
```

Comment:
Can we merge this with `storage.go` ?

Reply:
storage.go doesn't exist anymore :)

Reply:
my bad then! 
But I don't see the removal of the file in the PR.
I see a removed `reconcile.go`, but not `storage.go`. How is it removed ?

Reply:
It's renamed to `data_volumes.go`

Reply:
oh wow, got it now, the way the diff appears is very misleading (at least to me  :-) ) perfect then.